lms/djangoapps/verify_student/tasks.py

```diff
@@ -41,8 +41,9 @@ def on_success(self, retval, task_id, args, kwargs):
             'response_ok': boolean, indicating if the response was ok
             'response_text': string, indicating the response text in case of failure.
         """
-        user_verification_id = kwargs['user_verification_id']
-        user_verification = SoftwareSecurePhotoVerification.objects.get(id=user_verification_id)
+        from .models import SoftwareSecurePhotoVerification
```

Comment:
Why these separate imports? Why the model isn't being imported at the top?

Reply:
Actually there was an issue of circular import.

`models.py`  import `tasks.py`
and `tasks.py` import `SoftwareSecurePhotoVerification` model from `models.py` 

I have done the following steps to resolve that. 

1. `models.py` import `submit_request_to_ss` from `utils.py`. 
2. `utils.py` import `send_request_to_ss_for_user` from `tasks.py`
3. `tasks.py` shouldn't import `SoftwareSecurePhotoVerification` model to avoid circular import. So I am importing the model inside the function to avoid the error while during the app initialization.